samples/SkiaMultiplatformSample/README.md

```diff
@@ -0,0 +1,26 @@
+# Skia multiplatform samples
+
+## run iOS with Xcode
+ - Work's only on Mac with Intel CPU
```

Comment:
Building via Xcode does not work on M1. The problem has already been. The first attempt to fix it did not help.

For M1, assembly via AppCode will work

Reply:
So, it just doesn't/didn't work because of some not yet known issue in skiko build code/XCode/kotlin compiler/etc?

Let's rephrase this way then:
```
Works only on Mac with Intel CPU (for now)
```

Reply:
Yeah, good idea!